clerk-typedoc/clerk-react/use-session.mdx

```diff
@@ -17,21 +10,21 @@ This function returns a discriminated union type. There are multiple variants of
 | ------ | ------ | ------ |
 | `isLoaded` | `false` | A boolean that indicates whether Clerk has completed initialization. Initially `false`, becomes `true` once Clerk loads. |
 | `isSignedIn` | `undefined` | A boolean that indicates whether a user is currently signed in. |
-| `session` | `undefined` | The current active session for the user. |
```

Comment:
We shouldn't use the `active` semantics here anymore since it'll also include a `pending` session now. 

I'm updating the JSDocs on this PR: https://github.com/clerk/javascript/pull/6430